client/renderSDL/SDL_Extensions.h

```diff
@@ -71,6 +71,10 @@ SDL_Color toSDL(const ColorRGBA & color);
 	void convertToGrayscaleBpp(SDL_Surface * surf, const Rect & rect);
 	void convertToGrayscale(SDL_Surface * surf, const Rect & rect);
 
+	template<int bpp>
+	void convertToH2SchemeBpp(SDL_Surface * surf, const Rect & rect);
+	void convertToH2Scheme(SDL_Surface * surf, const Rect & rect);
```

Comment:
Not critical, but screen surface is guaranteed to be 4 bpp (32-bit), so such templates are no longer necessary.